.github/workflows/code-check.yml

```diff
@@ -168,9 +168,6 @@ jobs:
 
       - name: Setup Circuits
         uses: ./.github/actions/setup-circuits
-        with:
```

Comment:
If we're interacting with GH, which we do in the script, we might need to keep this, else we get rate limited very fast.